.github/workflows/bazelbuild.yml

```diff
@@ -0,0 +1,16 @@
+name: Bazel Build
+
+on: [pull_request, push]
+
+jobs:
+  macos:
+    name: Bazel
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Test iOS
+      run: bazelisk test //Tests:KronosTestsiOS --apple_platform_type=ios
+    - name: Test MacOS
+      run: bazelisk test //Tests:KronosTestsMacOS --apple_platform_type=macos
```

Comment:
I think you shouldn't need the `--apple_platform_type` flags here since you're calling specific test targets